server/sandbox_run.go

```diff
@@ -33,6 +33,14 @@ const (
 	// TODO: Remove this const once this value is provided over CRI
 	// See https://github.com/kubernetes/kubernetes/issues/47938
 	PodInfraOOMAdj int = -998
+
+	// KuberneteContainerNameLabel is same as the constant defined in k8s.
+	// TODO: This should be removed once we implemned the stats API.
+	KubernetesContainerNameLabel = "io.kubernetes.container.name"
+
+	// PodInfraContainerName is same as the leaky constant defined in k8s.
+	// TODO: This should be removed once we implement the stats API.
+	PodInfraContainerName = "POD"
```

Comment:
For this and `KubernetesContainerNameLabel` we can directly use kube types from upstream repo

Reply:
Switched to using k8s types.